app/views/thing_mailer/first_adoption_confirmation.html.erb

```diff
@@ -65,7 +66,7 @@
   <table width="100%" border="0" cellspacing="0" cellpadding="0">
     <tr>
       <td style="text-align: center;">
-        <%= image_tag image_url("adoptadrain-event.png"), :alt => "adopt-a-drain event", :title => "adopt-a-drain event" %>
+        <%= image_tag attachments['adoptadrain-event.png'].url, :alt => "adopt-a-drain event", :title => "adopt-a-drain event" %>
```

Comment:
How do we test this? I don't see a test in the code. We would have caught this earlier had there been one.

Reply:
In my next commit there is a test for this!